<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDrawDemoClass</class>
 <widget class="QWidget" name="QtDrawDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDrawDemo</string>
  </property>
  <layout class="QVBoxLayout" name="vLayout">
   <item>
    <widget class="QWidget" name="wgtTop" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>形状</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRect">
           <property name="text">
            <string>长方形</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSquare">
           <property name="text">
            <string>正方形</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEllipse">
           <property name="text">
            <string>椭圆</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCircle">
           <property name="text">
            <string>圆</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnArrow">
           <property name="text">
            <string>箭头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPen">
           <property name="text">
            <string>随意画</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>属性</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineWidth">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>线宽</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLineEdit" name="lineColor">
              <property name="text">
               <string>white</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>线颜色</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLineEdit" name="fillColor">
              <property name="text">
               <string>green</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>填充色</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="shadow">
           <property name="text">
            <string>阴影</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="round">
           <property name="text">
            <string>圆角</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sawtooth">
           <property name="text">
            <string>锯齿</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUse">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtCenter" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="wgtStatus" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>513</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtDrawDemo.qrc"/>
 </resources>
 <connections/>
</ui>
